src/components/shared/input/input.jsx

```diff
@@ -1,13 +1,37 @@
 import React from "react";
 import "./input.scss";
 import PropTypes from "prop-types";
+import { updateInputValue } from "../../../actions/actions";
+import { hot } from "react-hot-loader";
+import { connect } from "react-redux";
 
-const TextField = ({ className }) => (
-    <input type="text" className={className}/>
-);
+class TextField extends React.Component {
```

Comment:
Looks like functional component, why did you change it to class?

Reply:
Fixed